Update PR template, recommend `changelog: none` label,

tomalec opened the pull request:
### Changes proposed in this Pull Request:

Update PR template, recommend `changelog: none` label, instead of leaving completely empty section.

To match with the updated `yo grow` generator
https://github.com/woocommerce/grow/pull/38


### Screenshots:

<!--- Optional --->


### Detailed test instructions:
<!-- Add detailed instructions for how to test that this PR fixes the issue and confirm that it doesn't break any other features :) -->

1. Review[ PR template](https://github.com/woocommerce/google-listings-and-ads/pull/1613/files)


### Additional details:

<!--
Optional.
Enter a summary of all changes in this Pull Request, which will be added to the changelog if accepted.
Each line should start with change type prefix`(Fix|Add|…) - `, for example:
> Break - A change breaking previous API or functionality.
> Add - A new feature, function or functionality was added.
> Update - Big changes to something that wasn't broken.
> Fix - Took care of something that wasn't working.
> Tweak - Small change, that isn't actually very important.
> Dev - Developer-facing only change.
> Doc - Updated customer or developer facing documentation

Leave the "Changelog entry" header in place completely empty, without any summary if no changelog entry is needed.
If you remove the "Changelog entry" header, the title of Pull Request will be used as the changelog entry.
-->
### Changelog entry

> Dev - Update PR template.

tomalec committed:
Update PR template.

Recommend `changelog: none` label, instead of leaving complately empty section.

To match with the updated `yo grow` generator
https://github.com/woocommerce/grow/pull/38

mikkamp reviewed (approved):
Looks good to me, matches what's in the boilerplate.

I just couldn't help notice the wording "the title of Pull Request". I think that would sound better as one of these:
- the Pull Request title
- the title of the Pull Request

tomalec committed:
Fix grammar in the PR template

Addresses https://github.com/woocommerce/google-listings-and-ads/pull/1613#pullrequestreview-1061441446.

tomalec commented:
Thanks, updated here and in all the other PRs
